pkg/trait/trait_types.go

```diff
@@ -188,6 +188,22 @@ type ControllerStrategySelector interface {
 	ControllerStrategySelectorOrder() int
 }
 
+func isTrue(b *bool) bool {
```

Comment:
I'd suggest to move these methods into the `util.go` file.

Reply:
I mean the `pkg/trait/util.go` file, but if you think that's better in `pkg/util` package, that does it also.

Reply:
Ok, done all feedback. I should've come up with the elegant idea. Thanks.

Reply:
Ahhh, ok. But I think `BoolP` is useful outside of trait and traits are referred to from outside of trait so other bool pointer companion funcs should be also useful for the other parts as well.

Reply:
Right, I agree.